projects/plugins/jetpack/extensions/blocks/subscriptions/controls.js

```diff
@@ -56,22 +58,43 @@ export default function SubscriptionControls( {
 } ) {
 	return (
 		<>
+			{ subscriberCount > 1 && (
```

Comment:
Should we aim to display something when you only have 1 subscriber?

Reply:
I thought it was simpler to aim for two upwards and not deal with the singular. It'd be nice to add something about building an audience there for those cases instead of not showing anything.

Reply:
I updated this to >= 1. Hope that's okay. Although the first subscriber is usually the site owner.